packages/ie11-polyfills/package.json

```diff
@@ -11,5 +11,11 @@
   "devDependencies": {
     "ie11-custom-properties": "4.1.0"
   },
-  "license": "MIT"
+  "license": "MIT",
+  "beachball": {
+    "tag": "latest",
+    "disallowedChangeTypes": [
```

Comment:
Bit confused, why does this one get a special config?

Reply:
I updated repo-level beachball config. 
see "updated beachball config" section in my PR description :)